### ISSUE TITLE
Individual list pages

### ISSUE THREAD

**Issue by timgdavies** [OpenDataServices/org-ids#41]
It should be possible to visit a URL such as:

http://org-id.TLD/lists/GB-COH 

and get a page which includes all the meta-data about this list and ideally action-focussed UI that:

* [ ] Links to where to search for IDs;
* [ ] Provides details of how to suggest changes or updates;

**Comment by edugomez** [comment]
![org_ids_41](https://cloud.githubusercontent.com/assets/5618508/23122299/921bda32-f75b-11e6-8d65-c459748d8814.png)

![org_ids_41_page](https://cloud.githubusercontent.com/assets/5618508/23122303/963c86fc-f75b-11e6-87b6-929a782b7ef9.png)

**Comment by edugomez** [comment]
![org_ids_41](https://cloud.githubusercontent.com/assets/5618508/23122320/a65d4f76-f75b-11e6-98d1-40b0db1bbbc5.png)

**Comment by edugomez** [comment]
![org_ids_41_page](https://cloud.githubusercontent.com/assets/5618508/23122329/ad4a0ef0-f75b-11e6-9e5f-a0567beae946.png)

**Comment by edugomez** [comment]
> Provides details of how to suggest changes or updates

@timgdavies Could you elaborate more on that?

**Comment by timgdavies** [comment]
See #44

**Comment by timgdavies** [comment]
I've done some work on https://github.com/OpenDataServices/org-ids/tree/layout-edits to mock out what I would like these pages to look like.

I apologise for some of the worst crimes against bootstrap in the HTML there - but it's illustrative rather than for production.

Major things I've done:

* Add use of |linebreaks and |urlise filter to the output in most cases;
* Manually format the description and details on how to find the ID, and a section on how to use the code and identifier that is located;
* Add a block on how to suggest changes to the list, with a link to create a GitHub comment, and a link to the JSON source for people to fork and pull against;

I also explored, but didn't get time to implement, use of a markdown filter - as the descriptions are generally written in markdown that could be rendered. 

I've also made some tweaks to language on the results page, and in a few other places. 

If it is possible to clean up what I've done into something we wouldn't be ashamed to deploy - I would be grateful.
